Lint-Whoppers/nutshell
# Jb mj nyabar

# Description

Me and Michelle made the navbar and matched colors and other stuff.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ X] New feature (non-breaking change which adds functionality)

# Testing Instructions

All you should have to do is to open the branch and run NPM (or not) start. If you see our navbar and it is blue and blackish it should be great.



# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
